project/Http4sPlugin.scala

```diff
@@ -47,7 +47,16 @@ object Http4sPlugin extends AutoPlugin {
     addCompilerPlugin("org.spire-math" % "kind-projector" % "0.9.4" cross CrossVersion.binary),
 
     scalafmtVersion := "1.2.0",
-    scalafmtOnCompile := sys.env.get("TRAVIS").isEmpty
+    scalafmt in Test := {
+      (scalafmt in Compile).value
+      (scalafmt in Test).value
+      ()
+    },
+    test in (Test, scalafmt) := {
+      (test in (Compile, scalafmt)).value
+      (test in (Test, scalafmt)).value
+      ()
+    }
```

Comment:
I'm actually kind of surprised these aren't defaults.  If we're going to start making people run scalafmt manually, it's nice if we can give them just one command.